src/v/config/configuration.cc

```diff
@@ -779,7 +779,7 @@ configuration::configuration()
       0.0)
   , cloud_storage_upload_ctrl_min_shares(
       *this,
-      "compaction_ctrl_min_shares",
+      "cloud_storage_ctrl_min_shares",
```

Comment:
let's keep property name the same as variable name, this will limit number of names we need to track

Reply:
fixed that